kong-ngx-build

```diff
@@ -30,6 +31,14 @@ main() {
         NPROC=$2
         shift 2
         ;;
+      --pcre)
+        PCRE_VER=$2
+        shift 2
+        ;;
+      --pcre_sha)
+        PCRE_SHA=$2
+        shift 2
+        ;;
```

Comment:
I'm not sure why github is still showing these (the pcre stuff) changes they've been merged to master already